setup/functions.sh

```diff
@@ -47,6 +47,12 @@ function apt_get_quiet {
 	DEBIAN_FRONTEND=noninteractive hide_output apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confnew" "$@"
 }
 
+function apt_purge {
+	# Remove a bunch of packages.
+	PACKAGES=$@
+	apt_get_quiet --purge remove $PACKAGES
+}
+
```

Comment:
We don't need to support removing Nextcloud's dependencies from a system that it's already been installed on. You can remove this.

Reply:
Someone in these comments recommended this as an option, and I think it may be a pretty good one. For some folks, they're currently running MIAB with NextCloud, but would like to remove NextCloud at some future date. Why not also clean up the dependencies for them?